dataGeneration/generate-cosine-data-multi-entity.py

```diff
@@ -0,0 +1,294 @@
+'''
+
+Python script for ingesting sample data into OpenSearch index
+
+'''
+
+#{"index": {"_index":"host-cloudwatch","_id":"1177"}}
+#{"@timestamp":"2017-03-23T13:00:00","cpu":20.3, "memory":13,"host":"host1", "service": "service1"}
+
+import numpy as np
+from scipy.stats import uniform
+import datetime
+import time
+import random
+from random import Random
+from retry import retry
+import urllib3
+import concurrent.futures
+import argparse
+
+from opensearchpy import OpenSearch, RequestsHttpConnection
+from opensearchpy import helpers
+
+# https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
+urllib3.disable_warnings()
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument("-ep", "--endpoint", help="cluster endpoint", required=True)
+parser.add_argument("-i", "--index-name", help=" ",required=True)
+parser.add_argument("-shards", "--shards", type=int, help="The number of shards for the given index", required=True)
+parser.add_argument("-t", "--threads", type=int, help="The number of threads to be used for data ingestion, make sure given machine has enough", required=True)
+parser.add_argument("-bulk", "--bulk-size", type=int, default=3000, help="Number of documents per bulk request, default to 3000", )
+parser.add_argument("-ingest", "--ingestion-frequency", type=int, default=600, help="how often each respective document is indexed, for example the default is 600 seconds which equates to every 10 minutes")
+parser.add_argument("-p", "--points", type=int, default=1008, help="total number of points ingested, for example with 1008 points and a frequency of 600s, there will be 7 days of data")
+
+parser.add_argument('--security', action='store_true')
+parser.add_argument('--no-security', dest='security', action='store_false')
```

Comment:
Can we use something like `--security false` to replace `--no-security` ? But I'm ok for keep '--no-security'. Not a big problem.

Reply:
This was giving my issues because it was taking true/false param as a string as no type=boolean available. Thought it was less clean in code to add if "False", return false and elseif "True" return True. What do you think

Reply:
sure, sounds good, let's keep as is.